components/core/tests/test-EncodedVariableInterpreter.cpp

```diff
@@ -376,6 +376,47 @@ TEST_CASE("EncodedVariableInterpreter", "[EncodedVariableInterpreter]") {
         ));
     }
 
+    SECTION("Test multiple metching values") {
+        char const cVarDictPath[] = "var.dict";
+        char const cVarSegmentIndexPath[] = "var.segindex";
+        vector<string> var_strs = {"python2.7.3", "Python2.7.3", "PyThOn2.7.3", "PYTHON2.7.3"};
+        clp::VariableDictionaryWriter var_dict_writer;
+
+        var_dict_writer.open(cVarDictPath, cVarSegmentIndexPath, cVariableDictionaryIdMax);
+
+        vector<encoded_variable_t> encoded_vars;
+        vector<clp::variable_dictionary_id_t> var_ids;
+        clp::LogTypeDictionaryEntry logtype_dict_entry;
+        string const msg_template = "here is a string with a dictionary var: ";
```

Comment:
```suggestion
        string const msg_template{"here is a string with a dictionary var: "};
```
Let's use `{}` initializer whenever possible.